package.json

```diff
@@ -39,6 +39,7 @@
         "testJest": "jest"
     },
     "dependencies": {
+        "@ant-design/icons": "^4.7.0",
```

Comment:
Yes - react awesome query builder has a few UI libraries it can use and I liked this one quite a better than their react bootstrap integration which also seemed quite heavy. But I could try the bootstrap version as well if you think it's worth it?

Reply:
No, that's not what I'm saying - it's just that I think we can drop the `@ant-design/icons` dependency, keep the `antd` dependency, and everything will still work :)

Reply:
Ah gotcha :)